changelogs/unreleased/build-rpms-with-a-py311-venv.yml

```diff
@@ -0,0 +1,8 @@
+---
+description: The RPMs now install a Python 3.11 environment.
+issue-nr: 6024
+issue-repo: inmanta-core
+change-type: patch
+destination-branches: [master]
+sections:
+  feature: "{{description}}"
```

Comment:
Wouldn't upgrade note be more appropriate?

Reply:
Yes. I was doubting between upgrade note and feature. I went for feature because there is no real action required from the user. But maybe it's good to have it there anyway to increase the visibility of this change.